.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -371,17 +381,60 @@ jobs:
         # "test_conformance" target skips building "test_all" executable.
         ninja -C build-cts -k0 $( [ -n "$CTS_TESTS_TO_BUILD" ] && echo "$CTS_TESTS_TO_BUILD" || echo "test_conformance")
 
+    - name: Pack SYCL-CTS binaries
+      if: inputs.tests_selector == 'cts' && (success() || failure()) && inputs.cts_build_only == 'true'
+      run: tar -I 'zstd -9' -cf sycl_cts_bin.tar.zst -C ./build-cts/bin .
+
+    - name: Upload SYCL-CTS binaries
+      if: inputs.tests_selector == 'cts' && (success() || failure()) && inputs.cts_build_only == 'true'
```

Comment:
should we use similar vars to e2e mode, where we have `e2e_testing_mode` with `full`,`build-only` and `run-only`?